app/javascript/gobierto_dashboards/webapp/containers/costs/Distribution.vue

```diff
@@ -0,0 +1,160 @@
+            <h3>{{ labelTotalCost }}</h3>
+            <div class="metric">
+              {{ totalCost }}
+            </div>
+          </div>
+        </div>
+        <div class="pure-u-1-1 metric_box tipsit">
+          <div class="inner">
+            <h3>{{ labelCostPerInhabitant }}</h3>
+            <div class="metric">
+              {{ totalCostPerHabitant | money }}
+            </div>
+          </div>
+        </div>
+        <div class="pure-u-1-1 metric_box tipsit">
+          <div class="inner">
+            <h3>{{ labelInhabitant }}</h3>
+            <div class="metric">
+              {{ populationNumber }}
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        id="gobierto-dashboards-bubble-container"
+        class="pure-u-1 pure-u-lg-3-4"
+      >
+        <div class="vis-costs vis-bubbles" />
+        <div class="range-slider-costs">
+          <div class="range-slider-costs--container">
+            <div
+              v-for="item in years"
+              :key="item"
+              class="range-slider-costs--values"
+              :class="{ 'active-slider' : activeYear === item }"
+              @click="selectYearHandler(item)"
+            >
+              <span class="range-slider-costs--values-circle" />
+              <span class="range-slider-costs--values-text">{{ item }}</span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import { VisBubble } from "lib/visualizations";
+import { VueFiltersMixin } from "lib/shared";
+
+export default {
+  name: 'Distribution',
+  mixins: [VueFiltersMixin],
+  props: {
+    data: {
+      type: Array,
+      default: () => []
+    },
+    year: {
+      type: String,
+      default: ''
+    },
+    years: {
+      type: Array,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      labelDistribution: I18n.t("gobierto_dashboards.dashboards.costs.distribution") || "",
+      labelTotalCost: I18n.t("gobierto_dashboards.dashboards.costs.total_cost") || "",
+      labelInhabitant: I18n.t("gobierto_dashboards.dashboards.costs.inhabitants") || "",
+      labelCostPerInhabitant: I18n.t("gobierto_dashboards.dashboards.costs.cost_per_inhabitant") || "",
+      population: '',
+      populationNumber: '',
+      visBubblesCosts: null,
+      dataFilter: [],
+      activeYear: null
+    }
+  },
+  computed: {
+    totalCost() {
+      const total = this.dataFilter.reduce((accum,element) => accum + element.cost_total, 0)
+      return (total / 1000000).toFixed(1).replace(/\./, ',') + ' M€';
+    },
+    totalCostPerHabitant() {
+      return this.dataFilter.reduce((accum,element) => accum + element.cost_total, 0) / this.population
+    }
+  },
+  watch: {
+    year(newValue, oldValue) {
+      if (newValue !== oldValue) {
+        this.dataFilter = this.data.filter(element => element.year === newValue)
+        const [{
+          population: population
+        }] = this.dataFilter
```

Comment:
If you gonna get the first item of the array, why not using `find` instead of `filter`?